### PR TITLE
UINV-7 Add invoice line based on POL

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UINV-7
ability to create an invoice line that details the value and what is being paid for by the invoice.

## Approach

use find-po-line plugin to get lines and MCL to show lines list

#### TODOS and Open Questions

* tests

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/invoices/InvoiceDetails/InvoiceLines/AddInvoiceLinesAction.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <Pluggable
+      aria-haspopup="true"
+      dataKey="invoiceLines"
+      searchButtonStyle="default"
+      searchLabel={buttonLabel}
+      stripes={stripes}
+      type="find-po-line"
+      addLines={addLines}
+    >
+      <FormattedMessage id="ui-invoice.find-po-lin-plugin-unavailable" />
```
Comment (aliaksei-chumakou) [inline src/invoices/InvoiceDetails/InvoiceLines/AddInvoiceLinesAction.js]: `po-line`

**Comment by aliaksei-chumakou** [comment]
please check that you've added ui-plugin-find-po-line to platform-complete (snapshot branch)

**Comment by NikitaSedyx** [comment]
@aliaksei-chumakou I'll add it after merge PR from plugin repo, it's in my TODO

File: src/invoices/InvoiceDetails/InvoiceLines/InvoiceLines.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+class InvoiceLines extends Component {
+  static manifest = Object.freeze({
+    invoiceLines: {
+      ...invoiceLinesResource,
+      GET: {
+        params: {
+          query: (queryParams, pathComponents, resourceValues) => {
+            if (resourceValues.invoiceId && resourceValues.invoiceId.length) {
+              return `(invoiceId=${resourceValues.invoiceId})`;
```
Comment (aliaksei-chumakou) [inline src/invoices/InvoiceDetails/InvoiceLines/InvoiceLines.js]: please use `==`

**Pull request merged**